src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,83 @@
+.product-card {
+  width: 200px;
+  height: 408px;
```

Comment:
```suggestion

```
There is no need to hardcode the height for the card, usually, we have a different amount of content data in product cards